platforms/js/build_js.py

```diff
@@ -112,6 +112,7 @@ def get_cmake_cmd(self):
                "-DWITH_GPHOTO2=OFF",
                "-DWITH_LAPACK=OFF",
                "-DWITH_ITT=OFF",
+               "-DWITH_QUIRC=OFF",
```

Comment:
Why is it changed?

Reply:
quirc is a dependency of `QRCodeDetector` which has no wrappers yet.
https://github.com/opencv/opencv/blob/edacd91a274757d94be71dc78aaf24026125ae0b/modules/objdetect/include/opencv2/objdetect.hpp#L670
@allnes, Could you wrap it to python/java and add to the following file for JS bindings:
https://github.com/opencv/opencv/blob/edacd91a274757d94be71dc78aaf24026125ae0b/modules/js/src/embindgen.py#L118-L120